lib/ui/text.dart

```diff
@@ -833,47 +833,67 @@ class TextBox {
   String toString() => 'TextBox.fromLTRBD(${left.toStringAsFixed(1)}, ${top.toStringAsFixed(1)}, ${right.toStringAsFixed(1)}, ${bottom.toStringAsFixed(1)}, $direction)';
 }
 
-/// Whether a [TextPosition] is visually upstream or downstream of its offset.
+/// Disambiguates cases where an offset into a string in code could represent
```

Comment:
Disambiguates cases where a string offset could match two locations in the rendered string.